R/snapshot-auto.R

```diff
@@ -64,18 +69,14 @@ renv_snapshot_auto_enabled <- function(project) {
 
 }
 
-renv_snapshot_auto_update <- function(project) {
+renv_snapshot_auto_update <- function(project = renv_project_get() ) {
 
   # check for enabled
   if (!renv_snapshot_auto_enabled(project = project))
     return(FALSE)
 
-  # get path to project library
-  libpath <- renv_paths_library(project = project)
```

Comment:
`renv_snapshot_auto_enabled()` already checks that a library exists

Reply:
`renv_snapshot_auto_update()` also gets called in `renv_snapshot_task()`, which doesn't perform that validation, though, so we could run into an unexpected error if someone deleted the `renv/library` folder with automatic snapshots enabled.

Reply:
Oh, I'll delete the duplicated code the other way then.